modules/recording/recording/recording.tsx

```diff
@@ -143,14 +137,10 @@ const Recording = () => {
   const handleSave = () => {
     if (!(audioUrl && audioRef.current)) return;
 
-    const anchor = document.createElement("a");
-    anchor.href = audioUrl as string;
-
-    anchor.download = typeof audioUrl === "string" ? audioUrl : audioUrl[0];
-
-    document.body.appendChild(anchor);
-    anchor.click();
-    document.body.removeChild(anchor);
+    Download.downloadAudio(audioUrl as string).catch((error) => {
```

Comment:
Don't use `.catch()`  
Lets keep the it flat, use await.

Handle the error inside the try-catch of the function and this component can be more simple (No crash)

Every time we rehuse the function we need to handle the error, has sense they go inside